src/app/features/tech-record/components/tech-record-amend-vin/tech-record-amend-vin.component.spec.ts

```diff
@@ -72,31 +73,19 @@ describe('TechRecordChangeVrmComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  // describe('makeAndModel', () => {
-  //   beforeEach(() => {
-  //     expectedTechRecord = {systemNumber: 'foo', createdTimestamp: 'bar', vin: 'testVin'};
-  //     component.techRecord = expectedTechRecord;
-  //   });
-  //   // TODO: V3 PSV HGV TRL
-  //   // it('should should return the make and model', () => {
-  //   //   expect(component.makeAndModel).toBe(`${expectedTechRecord.chassisMake} - ${expectedTechRecord.chassisModel}`);
```

Comment:
Can we remove these commented out tests or do they need fixing?

Reply:
I mean this is removing them right?